### DISCUSSION TITLE
@designable/react、@designable/react-settings-form 不兼容 babel-plugin-import

### DISCUSSION THREAD

**Discussion by wwyx778** [alibaba/designable#137] [Q&A] [answered]
直接引入启动没问题，但是使用 babel-plugin-import 把 style.less 过滤时启动会报错，另外这两个包也没有打包后的 .css 文件。这个能处理一下吗？

![image](https://user-images.githubusercontent.com/28632663/143201610-c98c9042-c4a0-48f4-925a-aabd99e8610e.png)

**Comment by janryWang** [comment, marked as answer]
设计器不需要考虑按需打包的，没啥大的价值
